Quorum/checks/price_feed.py

```diff
@@ -75,10 +75,20 @@ def __check_price_feed_address(self, address: str, file_name: str) -> dict | Non
         """
         for provider in self.providers:
             if (price_feed := provider.get_price_feed(self.chain, address)):
+                
+                color = pp.Colors.SUCCESS
+                message = f"Found {address} on {provider.get_name()}\n"
+                message += str(price_feed)
+                if price_feed.proxy_address and price_feed.proxy_address.lower() != address.lower():
+                    message += f"Proxy address: {price_feed.proxy_address}\n"
+                if address.lower() != price_feed.address.lower():
+                    color = pp.Colors.FAILURE
+                    message += f"This is an implementation contract with a proxy address\n"
```

Comment:
Only this sentence should be printed in red

Reply:
It's not possible to do. and better to put the entire message to not miss it.

Reply:
But everything in red makes you think this is a failure

Reply:
It is a failure according to the ticket.

Reply:
If the contract uses the proxy address instead of the original they want it in red

Reply:
From ticket description, they want only the last line.
Please check this with Michael.

Reply:
ill merge it and if have a issue ill talk to him.

Reply:
Check with @MichaelMorami before merge.